### ISSUE TITLE
戦国BASARA3 宴 [BLJM60389] - Fatal error ingame, and when save is present game doesnt go beyond loading screen.

### ISSUE THREAD

**Issue by aethral** [RPCS3/rpcs3#12250]
When opening the game up, you get bombarded with
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/system/shader/ShaderPackage.mfx” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/system/shaderRev.arc” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/system/texture/sysfont_AM_NOMIP.tex” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/system/texture/DefaultCube_CM.tex” [1]
E SYS: '_sys_lwmutex_lock' failed with 0x80010005 : CELL_ESRCH [2]
E SYS: '_sys_lwmutex_lock' failed with 0x80010005 : CELL_ESRCH [3]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/rom/jpn/startup.arc” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_bdvd/PS3_GAME/USRDIR/resource/romstartup.xfw” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/sa/PS3/package.arc” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/sc/PS3/Basara/package.spkg” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/sc/PS3/Basara/Trophy/package.spkg” [1]
E SYS: 'sys_fs_open' failed with 0x80010006 : CELL_ENOENT, “/dev_bdvd/PS3_GAME/USRDIR/nativePS3/sc/PS3/Basara/Trophy/package.spkg” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/rom/jpn/init_ps3.arc” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/rom/jpn/sound/se_system_common.arc” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/rom/jpn/save.arc” [1]
E SYS: 'sys_fs_stat' failed with 0x80010006 : CELL_ENOENT, “/dev_hdd1/nativePS3/sc/PS3/Basara/Init/package.spkg” [1]
E SYS: 'sys_fs_open' failed with 0x80010006 : CELL_ENOENT, “/dev_bdvd/PS3_GAME/USRDIR/nativePS3/sc/PS3/Basara/Init/package.spkg” [1

if there's a save present.

Once you delete it, you can go ingame but you get a fatal error after while. First BGM just randomly cuts and then it'll just completly break.

**Comment by Megamouse** [comment]
missing log

**Comment by aethral** [comment]
[Log.txt](https://github.com/RPCS3/rpcs3/files/8965012/Log.txt)
Log included with the crash, crashes at "E RSX: nv406e::semaphore_acquire has timed out. semaphore_address=0x40300460" to be precise. First bgm cuts off then full freeze/crash.

**Comment by HIRO0132** [comment]
There is was similar error in BASARA4, but it has been improved by changing the PPU thread count setting on the debug tab to 1 In my environment.
Try it

**Comment by aethral** [comment]
One of the first things I tried as well since It worked in Tales of Xillia, but no luck here honestly.

**Comment by Megamouse** [comment]
Please try again

**Comment by aethral** [comment]
@[Megamouse](https://github.com/Megamouse)
Still getting this error when there's a save file present, basically forcing me to delete the save file 
[Save File Soft&Hard Lock.txt](https://github.com/RPCS3/rpcs3/files/9173690/Save.File.Soft.Hard.Lock.txt)


And just played for 30mins, same thing happening. First Music&Voice cuts off, then crash.
[Crash.txt](https://github.com/RPCS3/rpcs3/files/9173692/Crash.txt)

**Comment by digant73** [comment]
@aethral, @HIRO0132 is that still an issue?

**Comment by zananashi** [comment]
> [@aethral](https://github.com/aethral), [@HIRO0132](https://github.com/HIRO0132) is that still an issue?

Yes, game doesn't boot up still when save is present, and I think the resolution scaling is also not working

**Comment by PoserJpmetal** [comment]
aye bro i've done that shit already you just need to make resource folder after usrdir and name it resource fill it with blank notepads that you rename with romstartup.xfw that simple,and then for the spkg you just need make some folder,in native -> ps3 -> basara  you make folder in basara Init and trophy and then in basara folder you can copy the package.spkg to the folder init and trophy,and after all this theres still one bug if you restart the game the game will stuck,what do you have to do is download people save data decrypt that file name DAT0 the decrypt file you copy to to you save data you can replace it re run its and then enjoy the game.

-Sorry for my bad english hope this help

**Comment by digant73** [comment]
please an update on current build

**Comment by digant73** [comment]
please provide an update on current build and new log
